CITATION.cff

```diff
@@ -0,0 +1,30 @@
+cff-version: 1.2.0
+title: >-
+  Natural Language Toolkit
+message: >-
+  Please cite this software using the metadata from
+  'preferred-citation'.
+type: software
+authors:
+  - name: "NLTK Team"
+    email: "nltk.team@gmail.com"
+repository-code: "https://github.com/nltk/nltk"
+url: "https://www.nltk.org"
+license: Apache-2.0
+preferred-citation:
+  type: book
+  authors:
+    - given-names: Stephen
+      family-names: Bird
+      orcid: https://orcid.org/0000-0003-3782-7733
+    - given-names: Edward
+      family-names: Loper
+    - given-names: Ewan
+      family-names: Klein
+      orcid: https://orcid.org/0000-0002-0520-8447
+  title: >-
+    Natural language processing with Python: analyzing
+    text with the natural language toolkit
```

Comment:
Could we please have this in titlecase:
Natural Language Processing with Python: Analyzing Text with the Natural Language Toolkit

Reply:
Of course, done